modules/apigee_m10n_teams/src/TeamSdkControllerFactory.php

```diff
@@ -46,6 +47,17 @@ public function companyAcceptedRatePlanController(string $company_id): CompanyAc
     return $this->controllers[__FUNCTION__][$company_id];
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function companyTermsAndConditionsController(string $company_id): CompanyTermsAndConditionsController {
+    return new CompanyTermsAndConditionsController(
```

Comment:
All of the other controllers are using static caching for these controlelrs. See: `\Drupal\apigee_m10n_teams\TeamSdkControllerFactory::companyAcceptedRatePlanController`.